File: CordialDemo_Objective-C/CordialDemo_Objective-C/PushNotification/CordialPushNotificationHandler.m
//
//  CordialPushNotificationHandler.m
//  CordialDemo_Objective-C
//
//  Created by Yan Malinovsky on 7/29/19.
//

#import "CordialPushNotificationHandler.h"

@implementation CordialPushNotificationHandler 

- (void)appOpenViaNotificationTapWithNotificationContent:(NSDictionary * _Nonnull)notificationContent completionHandler:(void (^ _Nonnull)(void))completionHandler {
    completionHandler();
}

- (void)notificationDeliveredInForegroundWithNotificationContent:(NSDictionary * _Nonnull)notificationContent completionHandler:(void (^ _Nonnull)(void))completionHandler {
    completionHandler();
}

@end

File: CordialDemo/CordialDemo/PushNotification/CordialPushNotificationHandler.swift
//
//  CordialPushNotificationHandler.swift
//  CordialDemo
//
//  Created by Yan Malinovsky on 4/2/19.
//  Copyright © 2019 cordial.io. All rights reserved.
//

import Foundation
import CordialSDK

class CordialPushNotificationHandler: CordialPushNotificationDelegate {
    
    func appOpenViaNotificationTap(notificationContent: [AnyHashable : Any], completionHandler: @escaping () -> Void) {
        completionHandler()
    }
    
    func notificationDeliveredInForeground(notificationContent: [AnyHashable : Any], completionHandler: @escaping () -> Void) {
        completionHandler()
    }

    func apnsTokenReceived(token: String) {
        
    }
}
